src/components/ui/InfoBar.tsx

```diff
@@ -72,6 +72,7 @@ class InfoBar extends Component<IProps> {
                 lines={10}
                 scale={0.3}
                 color="#FFF"
+                // @ts-expect-error Property 'component' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<ReactLoader> & Readonly<LoaderProps>
```

Comment:
so, why do we need to introduce this `react-loader` at this point if we know that its to be replaced? Wasn't the codebase working without it?

Reply:
In general, it makes sense to install all `@types` dependencies which accompany the dependencies which are written in JS. Although we want to replace `react-loader`, it is unclear when this will happen so introducing it here makes sure that in case new usages of `react-loader` are used, they have to conform to the type definitions.